common/error.go

```diff
@@ -47,6 +47,8 @@ const (
 	CompatibilityAddCheck      Code = 10009
 	CompatibilityAlterCheck    Code = 10010
 	CompatibilityAlterColumn   Code = 10011
+	DeleteNoWhere              Code = 10012
```

Comment:
We can start from 10101 so future Compatibility error code can be adjacent.

And how about using a single error code?

StatementNoWhere